doc/cug.tex

```diff
@@ -5224,6 +5224,27 @@ \subsection{Task stdout And stderr Logs}
 files). For less internally complex tasks, however, the files referred
 to here will be complete task job logs.
 
+Some job submission methods, such as \lstinline=pbs=, redirect a job's stdout
+and stderr streams to a separate cache area while the job is running. The
+contents are only copied to the normal locations when the job completes. This
+means that \lstinline=cylc log= or the gcylc GUI will be unable to find the
+job's stdout and stderr streams while the job is running. Some sites with these
+job submission methods are known to provide commands for viewing and/or
+tail-follow a job's stdout and stderr streams that are redirected to these
+cache areas. If this is the case at your site, you can make use of the facility
```

Comment:
For clarity, how about:

If this is the case at your site, you can configure cylc to make use of the provided commands by adding ...